doc/cli/npm-access.md

```diff
@@ -6,7 +6,7 @@ npm-access(1) -- Set access level on published packages
     npm access public [<package>]
     npm access restricted [<package>]
 
-    npm access grant <read-only|read-write> <scope:team> [<package>]
+    npm access grant read-only|read-write <scope:team> [<package>]
```

Comment:
why not? without the angle brackets it might be confused for being pipes.

Reply:
No, `|` should never mean pipe in SYNOPSIS section of manuals or on-line usage message.

Reply:
However this change could make the 'or' items be misleading, so please also consider followings:

`npm access grant {read-only|read-write} <scope:team> [<package>]`
or
`npm access grant (read-only|read-write) <scope:team> [<package>]`